net-analyzer/synscan/synscan-5.02.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools eutils
+
+DESCRIPTION="A fast asynchronous half-open TCP portscanner"
+HOMEPAGE="http://www.digit-labs.org/files/tools/synscan/"
+SRC_URI="http://www.digit-labs.org/files/tools/${PN}/releases/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="kernel_FreeBSD kernel_linux"
+
+RDEPEND="net-libs/libpcap"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	mv "$S"/configure.in "$S"/configure.ac
+	eapply_user
+	eautoconf
+}
+
+src_configure() {
+	econf --prefix="${EPREFIX}"/usr
+}
+
+src_compile() {
+	local _target
+	use kernel_FreeBSD && _target=freebsd
+	use kernel_linux && _target=linux
+
+	emake ${_target}
+}
+
+src_install() {
+	dobin synscan sslog
+	dodoc AUTHORS README
```

Comment:
You should use `DOCS` instead of `dodoc` when possible. So you remove the `dodoc` line and define DOCS as: `DOCS=( AUTHORS README )` But, those are included in the default `DOCS`, see https://devmanual.gentoo.org/ebuild-writing/eapi/, so that's not even necessary.

Furthermore, you should rely on Make to do the installation instead of using `dobin` whenever possible. I took a look at Makefile.in, and it doesn't respect `DESTDIR`, so here's a patch fixing that: 
```
--- a/Makefile.in	2011-02-21 16:05:20.000000000 -0500
+++ b/Makefile.in	2019-03-20 09:55:57.656673219 -0400
@@ -65,10 +65,10 @@
 	${CC} ${LDFLAGS} ${CFLAGS} $< ${OBJS} ${LIBS} -o sslog
 
 install:
-	mkdir -p -m 755 ${bindir}
+	mkdir -p -m 755 $(DESTDIR)${bindir}
 	strip synscan
 	strip sslog
-	@INSTALL_PROGRAM@ synscan sslog ${bindir}
+	@INSTALL_PROGRAM@ synscan sslog $(DESTDIR)${bindir}
 
 clean:
 	rm -f ${PROJ} ${OBJS} core *.core
```
I emailed the patch to the upstream maintainers and CC'ed you.

You should:
 * Include that patch (using `PATCHES`)
 * remove `src_install` entirely

Reply:
ok.

Reply:
Candrews,

They also pre-stripped these binaries.I updated your patch and dropped them because of QA error.Is this ok for you?

 * QA Notice: Pre-stripped files found:
 * /usr/bin/sslog
 * /usr/bin/synscan

```
--- a/Makefile.in	2011-02-21 23:05:20.000000000 +0200
+++ b/Makefile.in	2019-03-20 18:28:46.812472904 +0300
@@ -65,10 +65,8 @@
 	${CC} ${LDFLAGS} ${CFLAGS} $< ${OBJS} ${LIBS} -o sslog
 
 install:
-	mkdir -p -m 755 ${bindir}
-	strip synscan
-	strip sslog
-	@INSTALL_PROGRAM@ synscan sslog ${bindir}
+	mkdir -p -m 755 $(DESTDIR)${bindir}
+	@INSTALL_PROGRAM@ synscan sslog $(DESTDIR)${bindir}
 
 clean:
 	rm -f ${PROJ} ${OBJS} core *.core
```

Reply:
That sounds good to me!